sys/cbor/cbor.c

```diff
@@ -1017,7 +1016,7 @@ static size_t cbor_stream_decode_at(cbor_stream_t *stream, size_t offset, int in
                     char buf[64];
                     struct tm timeinfo;
                     size_t read_bytes = cbor_deserialize_date_time(stream, offset, &timeinfo);
-                    strftime(buf, sizeof(buf), "%c", &timeinfo);
+                    strftime(buf, sizeof(buf), TIMESTRING_FORMAT, &timeinfo);
```

Comment:
Here, `"%c"` has been replaced by a date format. Was that an old remaining bug ?

Reply:
One of the warnings activated in this PR triggered on %c. I don't remember which warning, I just wanted to fix this in the least intrusive way possible since the module is deprecated and will be removed soon

Reply:
Sorry, I missed that it was `strftime` and not just `printf` which is why I did not understand.
And yes it silents the warning on native.

Reply:
Just for reference, if someone is interested, this date output changed:

* from: `Thu Jul  5 15:57:27 2018`
* to: `2018-07-05T15:57:48Z`